RELEASES.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## 0.19.0 (9-12-2015)
+- Switched to InfluxDB 9.4. The database schema was changed.
```

Comment:
Let's just say InfluxDB v0.9. That will let us rev up InfluxDB without a heapster release.